src/datetime.rs

```diff
@@ -1059,6 +1133,28 @@ mod tests {
                    Ok(EDT.ymd(2015, 2, 18).and_hms_micro(23, 59, 59, 1_234_567)));
     }
 
+    #[test]
+    fn test_rfc3339_opts() {
```

Comment:
Could you add a couple of these asserts to the docs for `to_rfc3339_opts`? Maybe just all 4 (pst/utc,true/false) of the `Millis` asserts?

Reply:
Yes, included in latest.